consensus-engine/src/overlay/tree_overlay/tree.rs

```diff
@@ -79,8 +79,12 @@ impl Tree {
     }
 
     pub(super) fn parent_committee_from_member_id(&self, id: &NodeId) -> Committee {
-        let Some(committee_id) = self.committee_id_by_member_id(id) else { return Committee::new(); };
```

Comment:
I think this needs rebasing. I think this fmt changes were merged in previous commits.